lms/djangoapps/mobile_api/groups/views.py

```diff
@@ -0,0 +1,141 @@
+            except Exception, e:
+                return Response({'error' : e.result['error']['message']}, status=status.HTTP_400_BAD_REQUEST)
+
+            return Response(app_groups_response)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+    def delete(self, request, *args, **kwargs):
+        if 'group_id' in kwargs:
+            post_args = {'method' : 'delete'}
+            graph = facebook.GraphAPI(facebook.get_app_access_token(_FACEBOOK_APP_ID, _FACEBOOK_APP_SECRET))
+            result = graph.request(_FACEBOOK_API_VERSION + _FACEBOOK_APP_ID + "/groups/" + kwargs['group_id'], post_args=post_args)
+            return Response(result)
+        else:
+            return Response({'error' : 'Missing group id'}, status=status.HTTP_400_BAD_REQUEST)
+
+
+
+class GroupsMembers(generics.CreateAPIView, mixins.DestroyModelMixin):
+    """
+    **Use Case**
+
+        An API to Invite and Remove members to a group
+
+    **Invite Example request**:
+
+        POST /api/mobile/v0.5/member/<group_id>/
+
+        Parameters: members : int,int,int... 
+
+
+    **Invite Response Values**
+
+        {"member_id" : success/error_message}       A response with each member_id and whether or not
+                                                    the member was added successfully. If the member was 
+                                                    not added successfully the Facebook error message is provided. 
+        
+    **Remove Example request**:
+
+        DELETE /api/mobile/v0.5/member/<group_id>/<member_id>
+
+    **Remove Response Values**
+
+        {"success" : "true"}                                    
+    """
+    authentication_classes = (OAuth2Authentication, SessionAuthentication)
+    permission_classes = (permissions.IsAuthenticated,)
+    serializer_class = serializers.GroupsMembersSerializer
+
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.DATA, files=request.FILES)
+        if serializer.is_valid():
+            graph = facebook.GraphAPI(facebook.get_app_access_token(_FACEBOOK_APP_ID, _FACEBOOK_APP_SECRET))
+            if 'group_id' in kwargs: 
+                url = _FACEBOOK_API_VERSION + kwargs['group_id'] + "/members"
+            member_ids = serializer.object['member_ids'].split(',')                        
+            response = {}
+            contains_error = False
+            for member_id in member_ids:
+                try:
+                    if 'success' in graph.request(url, post_args={'member' : member_id}): 
+                        response[member_id] = 'success'
+                except Exception, e:
+                    response[member_id] = e.result['error']['message']
+                    contains_error = True
+
+            status_code = status.HTTP_201_CREATED if contains_error else status.HTTP_200_OK
```

Comment:
I don't believe this API is conforming to the prescribed 201 response:
"The response SHOULD include an entity containing a list of resource characteristics and location(s) from which the user or user agent can choose the one most appropriate."

So we should always return a `200` here.